### ISSUE TITLE
`Office.context.mailbox.item.saveAsync` has a blank `value` field in JSON on success under new Outlook UI

### ISSUE THREAD

**Issue by ikexq** [OfficeDev/office-js#2099]
When `Office.context.mailbox.item.saveAsync` is called, it returns a blank `value` field when the call is successful ( `status : 'succeeded'` ). This only occurs when the new (beta) Outlook UI is toggled on/enabled.

## Your Environment
* Platform [PC desktop, Mac, iOS, Office on the web]: Mac
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 16.52
* Operating System: macOS Big Sur (11.5.2)
* Browser (if using Office on the web): N/A
* Other: New Outlook UI enabled.
 
## Expected behavior
On success, `Office.context.mailbox.item.saveAsync` should return a valid item ID (as documented [here](https://docs.microsoft.com/en-us/javascript/api/outlook/office.messagecompose?view=outlook-js-1.5&preserve-view=true#saveAsync_options__callback_))  in the `value` field e.g.
```
{
 value: ABCdEf123...A="",
 status: "succeeded"
}
```

## Current behavior
On success, `Office.context.mailbox.item.saveAsync` returns an empty value field  e.g.
```
{
 value: "",
 status: "succeeded"
}
```
**Note: This only seems to occur when the new UI is enabled.**

## Steps to reproduce
1. Create a new Outlook add in with an extension point that is triggered from compose e.g. 
    ` <extensionPoint xsi:type="MessageComposeCommandSurface">`
2. Function referenced by extension point should use `Office.context.mailbox.item.saveAsync`.
3. Start composing a new message.
4. Trigger your extension point.
5. View the response from `Office.context.mailbox.item.saveAsync` within the callback.

**Comment by gitcnd** [comment]
Didn't you mean this (**Compose**, not **Read**) ?

    <ExtensionPoint xsi:type="MessageComposeCommandSurface">

**Comment by ikexq** [comment]
> Didn't you mean this (**Compose**, not **Read**) ?
> 
> ```
> <ExtensionPoint xsi:type="MessageComposeCommandSurface">
> ```

Thank you, yes that is what I meant. I have updated the issue.

**Comment by exextoc** [comment]
We are not able to repro this issue at our end. We would like to collect the logs to understand the issue more. For that follow the following steps:
1. Repro the issue
2.  Click on Help button from Outlook menu
3. Click on Collect Diagnostics 
4. This will generate the dialog with incident ID

Share the Incident ID with us.

**Comment by ikexq** [comment]
Thanks, done. Incident ID is E5P2UTZA

**Comment by exextoc** [comment]
The fix for this issue is available in Mac Outlook versions >= 16.53.827.0 (16.53.21082700) and will be available in September Prod build. We recommend you either to update to the latest Insiders ring(>=  above mentioned version) and check Or wait till Prod build to release by mid of the month.

**Comment by gitcnd** [comment]
Cool!! Thank you SO much !!!

Quick question: it looks like basically all of the back-end support for MobileMessageComposeCommandSurface exists, but for some unknown reason, a decision not to let developers write mobile Outlook add-ins has been made?  How hard would it be to change that stance and let it work for us?  

Also - I notice that OWA still detects my phone as a phone, even when I enable "desktop mode" in chrome, and thus refuses to let me use MessageComposeCommandSurface as well.

What is the workaround for my customers who need my Add-in while mobile?  Do I have to ask them to move to gmail and stop using Microsoft?

Chris.

**Comment by ikexq** [comment]
> The fix for this issue is available in Mac Outlook versions >= 16.53.827.0 (16.53.21082700) and will be available in September Prod build. We recommend you either to update to the latest Insiders ring(>= above mentioned version) and check Or wait till Prod build to release by mid of the month.

Tested and confirmed as working on 16.53.21090501. Thanks for the prompt resolution!
